controllers/component_build_controller_test.go

```diff
@@ -761,6 +763,104 @@ var _ = Describe("Component initial build controller", func() {
 			deleteRoute(pacRouteKey)
 		})
 
+		It("should not submit initial build if pac secret is missing", func() {
```

Comment:
Looks like the test name is wrong

Reply:
why? I am removing pac secret and build will fail

Reply:
First, you're testing PaC not initial build. Second, it doesn't build, it's doing PaC unprovision.

Reply:
ok will rephrase :)